examples/nemo/scripts/utils.py

```diff
@@ -35,13 +35,15 @@
 
 ''' Utilities file to parallelise Nemo code. '''
```

Comment:
Not for this PR but I'm thinking it would be good to move as much of this as we can into `domain/nemo/` and then we can get it unit tested.

Reply:
I agree this is getting bigger than it should, in addition to `domain/nemo` I think the openmp collapsing can be an option to the generic omp_loop_trans/acc_loop_trans. The normalisation utility could also be a generic metatransformations once array2loop have been refactored, because it isn't NEMO specific anymore.